### PR TITLE
Fixing ImageMosaicCogOnlineTests due to S3 datasets no more available

### PR DESCRIPTION
**Pull request by dromagnoli**
Landsat 8 S3 datasets being referred by Some COG tests are no longer reachable, causing the test to fail.
Switching to use Sentinel2 datasets

### CODE REVIEW

File: modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicCogOnlineTest.java [hunk 1]
```diff
@@ -247,19 +254,18 @@ public void testEmptyMosaic() throws Exception {
     /** Checking time get extracted from remote URL too. */
     @Test
     public void testTimeDimensionMosaic() throws Exception {
-
         final File workDir = prepareWorkingDir("emptycog.zip", "timeMosaic", "");
         try (FileWriter out =
                 new FileWriter(
                         new File(TestData.file(this, "."), "/timeMosaic/datastore.properties"))) {
-            out.write("database=cogmosaic\n");
+            out.write("database=cogtimemosaic\n");
```
Comment (aaime) [inline modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicCogOnlineTest.java:261]: Is it going to create the db during the test?

See also the setup of the online test:
https://github.com/geotools/geotools/blob/main/.github/workflows/postgis_online.yml

Comment (dromagnoli) [reply modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicCogOnlineTest.java:261]: Yep. it's just H2 db

**Review by ianturton — APPROVED** [review]

**Comment by mprins** [comment]
should be backported to 26.x as well I think: https://github.com/geotools/geotools/runs/5633047158?check_suite_focus=true#step:8:506

**Comment by dromagnoli** [comment]
> should be backported to 26.x as well I think: https://github.com/geotools/geotools/runs/5633047158?check_suite_focus=true#step:8:506

Done

**Pull request merged**